CONTRIBUTING.md

```diff
@@ -1,45 +1,75 @@
 # How to contribute
 
-Thank you in your interest in the podman-py project.  We need your help to make it successful. 
+Thank you in your interest in the podman-py project.  We need your help to make
+it successful.
 
 You may also want to look at:
- - https://github.com/containers/libpod
- - https://podman.readthedocs.io/en/latest/Reference.html
- 
+
+- [podman](https://github.com/containers/podman)
+- [podman Reference](https://podman.readthedocs.io/en/latest/Reference.html)
+
 ## Reporting Issues
-Before reporting an issue, check our backlog of open issues to see if someone else has already reported it. If so, feel free to add your scenario, or additional information, to the discussion. Or simply "subscribe" to it to be notified when it is updated.
 
-If you find a new issue with the project we'd love to hear about it! The most important aspect of a bug report is that it includes enough information for us to reproduce it. So, please include as much detail as possible and try to remove the extra stuff that doesn't really relate to the issue itself. The easier it is for us to reproduce it, the faster it'll be fixed!
+Before reporting an issue, check our backlog of open issues to see if someone
+else has already reported it. If so, feel free to add your scenario, or
+additional information, to the discussion. Or simply "subscribe" to it to be
+notified when it is updated.
+
+If you find a new issue with the project we'd love to hear about it! The most
+important aspect of a bug report is that it includes enough information for us
+to reproduce it. So, please include as much detail as possible and try to
+remove the extra stuff that doesn't really relate to the issue itself. The
+easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
 ## Tools we use
- - https://www.pylint.org/
- - Python 3.6
- - You may need to use virtualenv to support Python 3.6
-   - https://virtualenv.pypa.io/en/latest/
- 
+
+- Python 3.6
+- [pylint](https://www.pylint.org/)
+- [black](https://github.com/psf/black)
+- [tox](https://tox.readthedocs.io/en/latest/)
+- You may need to use [virtualenv](https://virtualenv.pypa.io/en/latest/) to
+  support Python 3.6
+
 ## Testing
-Depending on the size of your PR we will expect at a minimum unit tests. 
+
+Depending on the size of your PR we will expect at a minimum unit tests.
```

Comment:
@mwhahaha Thanks for this work.

Given your coverage work and we're in the high 80's on coverage. How about putting a number on the unit test/coverage results? Won't merge if PR coverage falls below 85%?

Reply:
Sure we can raise that and document it